packages/design-systems/src/components/composite/Datagrid/useDataGrid.tsx

```diff
@@ -38,7 +38,7 @@ export const useDataGrid = <TData extends RowLike>({
   rowSelection,
   pageSizeOptions = [],
   enableDensity = false,
-  exportOptions = { enableCsvExport: false },
+  exportOptions = { enableCsvExport: false, omit: undefined },
```

Comment:
Can' we use `getDefaultOptions` in `Export` feature here?

```suggestion
  exportOptions = getDefaultOptions(),
```